sleap/gui/suggestions.py

```diff
@@ -275,6 +291,35 @@ def _velocity_video(
 
         return cls.idx_list_to_frame_list(frame_idxs, video)
 
+    @classmethod
+    def frame_chunk(
+        cls,
+        labels: "Labels",
+        videos: List[Video],
+        frame_from: int,
+        frame_to: int,
+        **kwargs,
+    ):
+        # check the validity of inputs, frame_from <= frame_to
+        if frame_from > frame_to:
+            return VideoFrameSuggestions.filter_unique_suggestions(labels, videos, [])
+
+        """Add consecutive frame chunk to label suggestion"""
+        proposed_suggestions = []
+        for video in videos:
+            # make sure when targeting all videos the from and to do not exceed frame number
```

Comment:
Nitpick: [google-style comments](https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar)
```suggestion
            # Make sure when targeting all videos the from and to do not exceed frame number
```